tests/accelerators/test_accelerator_connector.py

```diff
@@ -105,7 +106,7 @@ def test_accelerator_choice_ddp_slurm(setup_distributed_mock):
     class CB(Callback):
 
         def on_fit_start(self, trainer, pl_module):
-            assert trainer.use_ddp
+            assert trainer.accelerator_connector._distrib_type in (DistributedType.DDP, DistributedType.DDP_SPAWN)
```

Comment:
I think we can remove these lines completely.
We already have the check `assert isinstance(trainer.training_type_plugin, DDPPlugin)` below.